ga_resources/admin.py

```diff
@@ -11,6 +11,7 @@ class RenderedLayerAdminForm(ModelForm):
 
    class Meta:
       model = RenderedLayer
+      fields = '__all__'
```

Comment:
The new behavior is for security reasons and I'd prefer to take advantage of that rather than use the escape hatch to get the old, less secure behavior.